v3.0.0 + TypeScript Cannot find module 'redux-actions' from ...

The new ESM version fails to be imported by TypeScript.

Here's a reproducible comparison:

- v2.6.5: https://replit.com/@kaelig/ReduxActionsV2BugReport#index.ts
- v3.0.0: https://replit.com/@kaelig/ReduxActionsV3BugReportTS#index.ts

In the shell, run:

```console
tsc index.ts
```

- v2.6.5: compiles successfully
- v3.0.0:

```
~/ReduxActionsV3BugReportTS$ tsc index.ts
index.ts:1:30 - error TS2307: Cannot find module 'redux-actions' or its corresponding type declarations.

1 import { createAction } from 'redux-actions';
                               ~~~~~~~~~~~~~~~

Found 1 error in index.ts:1
```

<img width="1912" alt="Screenshot 2023-03-02 at 10 07 05 AM" src="https://user-images.githubusercontent.com/85783/222514188-6684623a-da0b-4178-9722-c414806f2629.png">

--- a/package.json
+++ b/package.json
@@ -18,7 +18,8 @@
   "module": "./dist/redux-actions.js",
   "exports": {
     ".": {
-      "import": "./dist/redux-actions.js"
+      "import": "./dist/redux-actions.js",
+      "default": "./dist/redux-actions.js"
     }
   },
   "keywords": [

